### PR TITLE
Updated the documentation for the ReadImage procedural interface.

### PR DESCRIPTION
**Pull request by zivy**
Change-Id: I87e8e7fb983b77a5a5e1f4aec22aec4fcefb6756

### CODE REVIEW

**Comment by blowekamp** [comment]
There is also are `ReadImage` method in the image series reader, that could use this information too.

**Comment by zivy** [comment]
Not really. The meta-data dictionary is lost when loading the series.

File: Code/IO/include/sitkImageFileReader.h [hunk 1]
```diff
@@ -69,7 +69,15 @@ namespace itk {
 
       std::string m_FileName;
     };
-
+  /**
```
Comment (blowekamp) [inline Code/IO/include/sitkImageFileReader.h]: A white space is needed before this doxygen comment.

**Comment by blowekamp** [comment]
Please rebase this commit onto the current master.

**Comment by zivy** [comment]
Done.

**Comment by blowekamp** [comment]
Looks great! Merging into next...

**Pull request merged**